### DISCUSSION TITLE
Motherboard Compatibility (Asus, AsRock, Gigabyte, MSI)

### DISCUSSION THREAD

**Discussion by zamazo** [Rem0o/FanControl.Releases#1429] [Q&A]
In the past few months, for various reasons, I've used AsRock, Asus, MSI, and Gigabyte AM4 mobos.
Seems AsRock and MSI ones work fine with this.
Asus = all sensors detected, but can't really control anything.
Gigabyte = 1/2 the sensors detected. Cannot control the detected ones.

Am just a sample size of 1 but does that sound right?
    
    
      

---------------edit: short summary below---------------

Motherboards that seem to work:
Asus ROG X670E Crosshair Extreme
AsRock B550M-ITX/AC
Msi MPG X570S Carbon MAX 

Motherboards that did not work:
Asus Prime x570 pro
Gigabyte X570S AORUS Master

**Comment by Rem0o** [comment]
It's not a brand thing. The software does work on most/some Asus/Gigabyte motherboards.

**Reply by Fluff973** [reply]
FWIW, it does work with my Asus ROG X670E Crosshair Extreme board.

**Reply by zamazo** [reply]
@Fluff973 Thanks! I'm actually looking at Asus's X670E-A or X670E-PRO for my next board. So that's encouraging. 
Can I ask if you've tried turning on manual control and briefly setting fans, including the CPU/CPU_OPT to 0 and seeing if it actually lets the fans stop fully?

**Reply by Fluff973** [reply]
Yes, both manual control and 0% RPM (fan stop) mode work correctly with my Noctua 120 and 140mm fans on CPU/CPU_OPT headers and other case headers I've tried. No problems at all.

Having said that, some fans don't support 0 RPM mode. For example, the Lian Li fans which originally came with my Lian Li O11 Air mini case don't stop fully, but that's a characteristic of those fans, not the motherboard's (or FanControl's) fault.

**Reply by zamazo** [reply]
Thanks man! That really helps a lot. 

And yea I know. 
But unfortunately my Asus prime X570 pro is just not compatible. 
Works fine same setup but with msi carbon x570s and asrock tho.

**Comment by zamazo** [comment]
Can we get a 'known works with' or a 'known not working' list going ya think?

Had to return the Gigabyte x570s since it was incompatible.

**Reply by Rem0o** [reply]
There is so many, and even If I maintained a list the best I could, I couldn't guarantee compatibility for the the hardware listed.

**Reply by zamazo** [reply]
I meant a community maintained list, would be insane to personally test that much :)
But then again, not sure how you'd do that without a million pull requests or something.
Maybe just a thread for posting what worked/didn't work?

**Reply by zamazo** [reply]
@Rem0o can I at least ask what mobos you personally use and thus have confirmed to be working 100% with this?
Any AM5 ones?

**Comment by pookguy88** [comment]
is there a hardware fan controller that works? I'm thinking to just get a fan controller that works 100% with fancontrol

**Reply by Rem0o** [reply]
Haven’t tested any myself.

**Comment by BeerMan81** [comment]
I have aAORUS RGB FAN COMMANDER that came with my motherboard - [Gigabyte x570 extreme](https://www.gigabyte.com/Motherboard/X570-AORUS-XTREME-rev-10#kf). It detects and controls pump and one fan which ar connected directly to the MOBO, but anything connected to the commander does not work.
